xgboost_ray/main.py

```diff
@@ -882,12 +882,14 @@ def _create_placement_group(cpus_per_actor, gpus_per_actor,
     if ready:
         logger.debug("Placement group has started.")
     else:
-        raise TimeoutError("Placement group creation timed out. Make sure "
-                           "your cluster either has enough resources or use "
-                           "an autoscaling cluster. Current resources "
-                           "available: {}, resources requested by the "
-                           "placement group: {}".format(
-                               ray.available_resources(), pg.bundle_specs))
+        raise TimeoutError(
+            "Placement group creation timed out. Make sure "
```

Comment:
Let's print the timeout here

```suggestion
            "Placement group creation timed out after ... seconds. Make sure "
```